receiver/signalfxreceiver/receiver.go

```diff
@@ -232,6 +247,9 @@ func (r *sfxReceiver) failRequest(
 		}
 	}
 
+	// Use the same pattern as strings.Builder String().
+	msg := *(*string)(unsafe.Pointer(&jsonResponse))
```

Comment:
Looks scary :) what does it do?

Reply:
avoids copying the contents of the `[]byte` it is stolen from here: https://golang.org/src/strings/builder.go?s=1379:1412#L45